ux.symfony.com/README.md

```diff
@@ -2,48 +2,63 @@
 
 Source code for [ux.symfony.com](https://ux.symfony.com).
 
-## Contributing
+## Installation
```

Comment:
as a dev who wants to contribute/install locally, why not create a bare raw makefile with all such instructions in it?
i dont see the value of adding them in the readme :s

Reply:
I did not "add" them you know ? :) 

For this README i'm not sure to see what's "wrong" to have a bit of documentation / getting started. 

But If you want to add / use a Makefile i'm sure no one will block you :)

Reply:
I mean to help a guys who wants to contribute in this sub project, it surely know symfony and related logic of install
so the added value to describe in the readme is low (to me of course ^^)
perhaps a bare raw makefile and tell the dev to just go make install may be better ;)